NPM errors when saving changes to theme sources.

<!--- STOP! Before you open an issue please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created. -->
<!--- SECURITY DISCLOSURE: If this is a security disclosure please follow the guidelines in CONTRIBUTING.md. This helps keep folks from accidentally releasing vulnerabilities before the maintainers get a chance to fix the issue. -->

### Expected Behaviour
Changes compile and can be previewed on port localhost:3000

### Actual Behaviour
All changes after the first save throw an NPM error

### Reproduce Scenario (including but not limited to)
Making a secondary change to theme-sources after successful first change.

#### Steps to Reproduce
1. In local open theme sources
2. In theme source folder, run npm run live
3. Wait for browser to open
4. Make change in src/site/_variables.scss
5. Save changes
6. Wait for npm to pick up changes
7. Re-fresh to see changes on localhost:3000
8. Go through workflow steps 4 - 7
9. npm throws an error

#### Platform and Version
Microsoft Windows 10 Pro
OS Version: 10.0.19042 N/A Build 19042

#### Sample Code that illustrates the problem

#### Logs taken while reproducing problem

> aem-site-template-basic-theme@0.0.1 compile C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources
> webpack -d --env dev --config ./webpack.dev.js

Error: ENOTEMPTY: directory not empty, rmdir 'C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\dist\css'
    at Object.rmdirSync (fs.js:850:3)
    at rmkidsSync (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\rimraf\rimraf.js:364:25)
    at rmdirSync (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\rimraf\rimraf.js:342:7)
    at Function.rimrafSync [as sync] (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\rimraf\rimraf.js:312:9)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\del\index.js:65:11
    at Array.map (<anonymous>)
    at module.exports.sync (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\del\index.js:57:40)
    at CleanWebpackPlugin.removeFiles (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\clean-webpack-plugin\dist\clean-webpack-plugin.js:198:37)
    at CleanWebpackPlugin.handleInitial (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\clean-webpack-plugin\dist\clean-webpack-plugin.js:135:10)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\clean-webpack-plugin\dist\clean-webpack-plugin.js:84:16
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:7:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at Compiler.emitAssets (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compiler.js:491:19)
    at onCompiled (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compiler.js:278:9)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compiler.js:681:15
    at eval (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:18:1)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\ts-loader\dist\after-compile.js:28:9
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:14:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compiler.js:678:31
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:4:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compilation.js:1423:35
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:4:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compilation.js:1414:32
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:4:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compilation.js:1409:36
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:4:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compilation.js:1405:32
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:4:1)
    at AsyncSeriesHook.lazyCompileHook (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\tapable\lib\Hook.js:154:20)
    at Compilation.seal (C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compilation.js:1342:27)
    at C:\aem-demo-machine-master\demos\qsc\basic-theme\theme-sources\node_modules\webpack\lib\Compiler.js:675:18
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aem-site-template-basic-theme@0.0.1 compile: `webpack -d --env dev --config ./webpack.dev.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the aem-site-template-basic-theme@0.0.1 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\cochrane\AppData\Roaming\npm-cache\_logs\2021-05-13T14_51_35_283Z-debug.log

--- a/theme/package.json
+++ b/theme/package.json
@@ -8,15 +8,13 @@
     "scripts": {
         "build": "webpack --config ./webpack.prod.js",
         "compile": "webpack --config ./webpack.dev.js",
-        "proxy": "aem-site-theme-builder live",
-        "watch": "chokidar \"src\" -c \"npm run compile\"",
-        "live": "npm-run-all -p watch proxy"
+        "proxy": "aem-site-theme-builder proxy",
+        "live": "npm-run-all -p compile proxy"
     },
     "devDependencies": {
         "@adobe/aem-site-theme-builder": "5.0.0",
         "autoprefixer": "^10.3.0",
         "browser-sync": "^2.26.13",
-        "chokidar-cli": "^2.1.0",
         "clean-webpack-plugin": "^3.0.0",
         "copy-webpack-plugin": "9.0.1",
         "cross-var": "^1.1.0",

--- a/theme/webpack.dev.js
+++ b/theme/webpack.dev.js
@@ -11,11 +11,13 @@ const { merge }         = require('webpack-merge');
 const common            = require('./webpack.common.js');
 const path              = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const BrowserSyncPlugin = require('browser-sync-webpack-plugin')
 
 const SOURCE_ROOT = __dirname + '/src';
 
 module.exports = merge(common, {
    mode: 'development',
+   watch: true,
    devtool: 'inline-source-map',
    performance: { hints: "warning" },
    devServer: {
@@ -24,5 +26,12 @@ module.exports = merge(common, {
          context: ['/content', '/etc.clientlibs'],
          target: 'http://localhost:4502',
       }]
-   }
+   },
+   plugins: [
+      new BrowserSyncPlugin({
+        host: 'localhost',
+        port: 3000,
+        proxy: 'http://localhost:7000/'
+      })
+   ]
 });

